### DISCUSSION TITLE
Is rtx4060 enough to complete this project?

### DISCUSSION THREAD

**Discussion by MichaelChoum** [leggedrobotics/viplanner#50] [General]
Hello, thanks for your nice project! It's really a great job! I wonder whether it is enough to accomplish this project with rtx4060 only？

**Comment by pascal-roth** [comment]
Hi,

Thanks a lot for using our work. For the data collection, 8 GB can be though due to the Isaac Sim requirements (https://docs.omniverse.nvidia.com/isaacsim/latest/installation/requirements.html). As I don't a GPU in that size, I sadly cannot test it. 

Feel free to try it and then share here if it worked.
